onmt/inputters/dataset_base.py

```diff
@@ -2,19 +2,12 @@
 
 from itertools import chain
 from collections import Counter
-import codecs
 
 import torch
 from torchtext.data import Example, Dataset
 from torchtext.vocab import Vocab
 
 
-# several data readers need optional dependencies. There's no
-# appropriate builtin exception
-class MissingDependencyException(Exception):
-    pass
-
-
 class DatasetBase(Dataset):
```

Comment:
Is `DatasetBase` still an appropriate name for this class? It's just a dataset now, isn't it?

Reply:
I agree but then IMO the file needs renamed. I didn't want to blow up the diff by doing that now since there are actual changes to review here.